Modules/_asynciomodule.c

```diff
@@ -3623,6 +3623,14 @@ _asyncio_all_tasks_impl(PyObject *module, PyObject *loop)
         Py_DECREF(item);
     }
     Py_DECREF(eager_iter);
+
+    /* Check if loop ended because of exception in PyIter_Next */
```

Comment:
```suggestion
    // Handle a possible exception in PyIter_Next
```

Reply:
Remove them or make them all as you suggest?

Reply:
I would remove them personally, but if you want to keep them either your or my suggestion (now that I look at it, mine is not really better =/)

Reply:
Probably it does not make sense to keep them if there will be follow up PR with replacement of `PyIter_Next` with `PyIter_NextItem`

Reply:
Let's remove them then.

Reply:
Done